tests/util/aws/test_common.py

```diff
@@ -50,6 +49,15 @@
                 'S3Bucket': BUCKET,
                 'S3Prefix': PREFIX,
                 'S3Region': REGION}]
+MOCK_BOTO_CLIENT = Mock()
+response = {
+    'Credentials': {
+        'AccessKeyId': 'mock_access_key_id',
+        'SecretAccessKey': 'mock_secret_access_key',
+        'SessionToken': 'mock_session_token'
```

Comment:
As above - let's use Faker to inject randomized strings, instead of hard-coded ones. If we're not testing for a specific value, a randomized value may help us spot more bugs by not having embedded assumptions about the structure of the string.  (e.g. Access Key Ids never have underscores. So, if we're testing using this string, it'll encourage us to create tests based on an incorrect set of requirements.)